homeassistant/components/sensor/sht31.py

```diff
@@ -0,0 +1,152 @@
+"""
+Support for Sensirion SHT31 temperature and humidity sensor.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.sht31/
+"""
+
+from datetime import timedelta
+import logging
+import math
+
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import (
+    TEMP_CELSIUS, CONF_NAME, CONF_MONITORED_CONDITIONS)
+from homeassistant.exceptions import HomeAssistantError
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+from homeassistant.helpers.temperature import display_temp
+from homeassistant.const import PRECISION_TENTHS
+from homeassistant.util import Throttle
+
+
+REQUIREMENTS = ['Adafruit-GPIO==1.0.3',
```

Comment:
Does this library somehow place a lock on the GPIO interface? In other words: would using this platform together with other GPIO platforms like DHT22 or [RPi GPIO](https://home-assistant.io/components/binary_sensor.rpi_gpio/)?

Reply:
As far as I'm aware there isn't any kind of lock on GPIO. Not sure thought. All I can say is that I've just tried using DHT22 and SHT31 together and they work.